usecases/replica/coordinator.go

```diff
@@ -28,6 +28,8 @@ type commitOp[T any] func(_ context.Context, host, requestID string) (T, error)
 // readOp defines a generic read operation
 type readOp[T any] func(_ context.Context, host string) (T, error)
 
+type readOp2[T any] func(_ context.Context, host string, fullRead bool) (T, error)
```

Comment:
A more contextual name would be helpful here

Reply:
Sure
readOp is there just for backward comp 
It will be removed and replaced by readOp2